locales/en-US/app.ftl

```diff
@@ -1,8 +1,62 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 ### Localization for the App UI of Profiler
 
-# Naming convention for l10n IDs: "module-ComponentName--string-summary".
+# Naming convention for l10n IDs: "ComponentName--string-summary".
 # This allows us to minimize the risk of conflicting IDs throughout the app.
-# Please sort alphabetically by (module name, component name). For each module,
+# Please sort alphabetically by (component name).
 # keep strings in order of appearance.
 
-intro = Web app for Firefox performance analysis
+## The following feature names must be treated as a brand. They cannot be translated
+
+-firefox-brand-name = Firefox
+-profiler-brand-name = Firefox Profiler
+
+## Translation Strings
+
+AppHeader--app-header = <header>{ -profiler-brand-name }</header> – <subheader>Web app for Firefox performance analysis</subheader>
+AppHeader--github-icon =
+    .title = Go to our git repository (this opens in a new window)
+
+FooterLinks--legal = Legal
+FooterLinks--Privacy = Privacy
+FooterLinks--Cookies = Cookies
+
+Home--profiler-motto = Capture a performance profile. Analyze it. Share it. Make the web faster.
+Home--documentation-button = Documentation
+Home--instructions-title = How to view and record profiles
+Home--instructions-content = 
+    Recording performance profiles requires <a>Firefox</a>.
+    However, existing profiles can be viewed in any modern browser.
+
+Home--additional-content-title = Load existing profiles
+Home--additional-content-content = You can <strong>drag and drop</strong> a profile file here to load it, or:
+Home--upload-from-file-input-button = Load a profile from file
+Home--upload-from-url-button = Load a profile from a URL
+Home--compare-recordings-info = You can also compare recordings. <a>Open the comparing interface.</a>
+Home--recent-uploaded-recordings-title = Recent uploaded recordings
+Home--record-instructions = 
+    To start profiling, click on the profiling button, or use the
+    keyboard shortcuts. The icon is blue when a profile is recording. 
+    Hit <kbd>Capture Profile</kbd> to load the data into profiler.firefox.com.
+
+Home--record-instructions-start-stop = Stop and start profiling
+Home--record-instructions-capture-load = Capture and load profile
+Home--load-from-url-submit-button =
+    .value = Load
+
+ListOfPublishedProfiles--publishedProfilesLink =
+    .title = Click here to load profile { $smallProfileName }
```

Comment:
Please always add comments for variables
https://firefox-source-docs.mozilla.org/l10n/fluent/review.html#comments

Same comment applies to strings appearing later in the file.